.deploy/prod-gcp.json

```diff
@@ -14,6 +14,7 @@
     "https://k9-inntektsmelding.intern.nav.no"
   ],
   "dbTier": "db-custom-1-3840",
+  "enableDBAudit": "false",
```

Comment:
Usikker på om denne togglen vil fungere som tiltenkt.

I handlebars så anses en ikke-tom string som true uansett verdien.
Ville prøvd å fjerne fnuttene og brukt boolean som verdi.

Samme i dev-gcp.json.

Reply:
Du kan bruke dette lokalt og se hva som faktisk settes: https://doc.nais.io/operate/cli/reference/validate/?h=validat#verbose-output

Reply:
jeg lurer på man må ha strenger i json fila. jeg gjorde tilsvarende et annet sted, men godt tips, jeg kan teste.

Reply:
du hadde rett! 🏆  Den ble true selv om jeg skrev false 🙌  godt sett!

Reply:
Ja, jeg tror det kan være pga. nais-deploy når den parser. Det var noen år siden jeg var bort i det. Kan være det er fikset nå.

For å være helt sikker kan evt. dra ut hele audit logging som objekt i variabel-filene. Deretter sjekke om objektet fines med en if-block. Dersom den finnes sette verdiene fra objektet.